fluentd/ECS/readme.md

```diff
@@ -0,0 +1,5 @@
+Coralogix Fluentd AWS ECS Image: This folder contains the docker file for the image Coralogix promotes for ECS usecases. fluent.conf - is the default conf, for EC2 usecases. firelens.conf - a configuration ment specificly for Fargate usecases, must be enabled by configuring via Json.
```

Comment:
Typo + add firelens context

Reply:
added:
the Json config + added that Firelens will use it's own config if not specified by the Json